sycl/test-e2e/AOT/double.cpp

```diff
@@ -1,9 +1,10 @@
 // This test ensures that a program that has a kernel
 // using fp64 can be compiled AOT.
 
-// REQUIRES: ocloc
+// REQUIRES: ocloc, opencl-aot
 // RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_tgllp -o %t.tgllp.out %s
 // RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_cfl -o %t.cfl.out %s
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_x86_64 -o %t.x86.out %s
```

Comment:
i think we need a `REQUIRES` for `any-device-is-cpu`, i think we can only compile for `spir64_x86_64` if the OCL CPU RT is installed as per [this](https://github.com/intel/llvm/blob/56e88d591c52a978abdd5e4279853311cae4a55e/opencl/opencl-aot/README.md?plain=1#L11), i just fixed an issue related to that for another test [here](https://github.com/intel/llvm/commit/781b00c19d4ce266877c2ca5a4d7d30b7a28e8ce).

Reply:
Also `REQUIRES: opencl-aot`

Reply:
[b494344](https://github.com/intel/llvm/pull/14590/commits/b494344f8d01e1f1bf0988be6d46008b0dbcedc5)